release notes/v0.39.0.md

```diff
@@ -0,0 +1,32 @@
+k6 v0.39.0 is here! :tada: It's a small release that includes a bunch of bugfixes and minor enhancements. A lot of our focus was on some upcoming big changes, details for which you can read at the end of this document!
```

Comment:
```suggestion
k6 v0.39.0 is here! :tada: It's a small release that includes a bunch of bugfixes and minor enhancements. A lot of our focus was on some upcoming big changes. You can read about what's next at the end of this document.
```

Reply:
Resolved by the latest [force push](https://github.com/grafana/k6/compare/9d682d82437985265e135cd6a24080be0b007dbd..6f763113a21481f8b3d72ac5496366cd5ffbd982)